class/practice3.2/include/Triangle.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <string>
+class Triangle {
+public:
+
+    Triangle(const double& first, const double& second, const double& third);
+
+    std::string getArea();
```

Comment:
use std::optional<double> as return type. It requires #include <optional> header in this file